### ISSUE TITLE
FY23 OSE: Fee Schedule Scoping Meeting and Review (On PROD)

### ISSUE THREAD

**Issue by Nadin-Nader** [cityofaustin/atd-data-tech#9064]
##  This issues is to start the Scoping of OSE's Fee schedule. This is also to review the changes wanted and needed to see if they are doable

**Comment by Nadin-Nader** [comment]
Maria from OSE sent an email with their Fees,
FY23 Fees:
https://app.zenhub.com/files/140626918/67767b83-c057-4d07-8680-a899d0a41b27/download

**Comment by SubhaShroff** [comment]
@Nadin-Nader @TracyLinder  @vschandramouli 
1. GL account number - In the excelsheet the AC number is given as - 5125 2400 9100 4502. Please confirm if it has to be in this format - 5125-2400-9100-4502-0000-0000-0000-0000-0000.

**Comment by TracyLinder** [comment]
Hi @SubhaShroff , those are great questions. We're not ready to begin fee updates yet; Vish will be creating the documentation this week. We can discuss at tomorrow's team meeting.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Analysis is in Progress

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - DEV & TEST DP submitted. Kindly approve and create task id for me.

Commit ID: [6e57caebe058](http://coaamandadevops02.austintexas.gov/R5:6e57caebe05840677eab5565f180797765a322a7)

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Tested in Localhost here is the screen shot with newly added fee

![image](https://user-images.githubusercontent.com/44907715/191831031-f9e0e387-75e9-4233-b86d-bc95e7b705e4.png)

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been sent to the Devops CM Team to deploy to Sandbox DEV and TEST under **_SCTASK0250678_**

**Comment by Nadin-Nader** [comment]
@sree this has been deployed to Sandbox, DEV and TEST Please UAT so we can have Users UAT on TEST

**Comment by TracyLinder** [comment]
### UAT Feedback 

- Update 50130 fee description to Street Event Safety Inspection Fee _(will do in a separate card)_

**Comment by Nadin-Nader** [comment]
The Name change will be done after all fees go to PROD but OSE Fees have been approved for PROD

**Comment by TracyLinder** [comment]
Created #10345 for the fee name update.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder  - PROD DP of #9064 submitted. Kindly approve and create task id for me. 

Commit ID: 6e57caebe058

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan  This has been sent to the DevOps CM Team to be deployed to **_PROD_** under **SCTASK0252214**

**Comment by Nadin-Nader** [comment]
This has been deployed to PROD

**Comment by Nadin-Nader** [comment]
Sent a message to Maria asking if her fees were good to go. Waiting on response

**Comment by Nadin-Nader** [comment]
From Maria......"So far so good, the only update was that Street Event Inspection Fee to Street Event Safety Inspection Fee"

I informed her that i would close the card and that the change mentioned is already a card.
